Jenkinsfile

```diff
@@ -108,6 +108,7 @@ pipeline {
   options {
     durabilityHint('PERFORMANCE_OPTIMIZED')
     buildDiscarder(logRotator(numToKeepStr: '15', artifactDaysToKeepStr: '15'))
+    timeout(time: 15, unit: 'HOURS')
```

Comment:
:+1

Reply:
We should also create more (smaller) test groups, [a paste of some test groups from Jakarta EE 8 results for WildFly](https://github.com/wildfly/certifications/blob/EE8/WildFly_19.0.0.Final/jakarta-full-platform.adoc)):
```
Test name: jms_core
[javatest.batch] Completed running 2379 tests.
[javatest.batch] Number of Tests Passed      = 2379
[javatest.batch] Number of Tests Failed      = 0
[javatest.batch] Number of Tests with Errors = 0

Test name: jms_core20
[javatest.batch] Completed running 852 tests.
[javatest.batch] Number of Tests Passed      = 852
[javatest.batch] Number of Tests Failed      = 0
[javatest.batch] Number of Tests with Errors = 0

Test name: jms_ee
[javatest.batch] Completed running 207 tests.
[javatest.batch] Number of Tests Passed      = 207
[javatest.batch] Number of Tests Failed      = 0
[javatest.batch] Number of Tests with Errors = 0

Test name: jms_ee20
[javatest.batch] Completed running 72 tests.
[javatest.batch] Number of Tests Passed      = 72
[javatest.batch] Number of Tests Failed      = 0
[javatest.batch] Number of Tests with Errors = 0

Test name: jpa_core_StoredProcedureQuery
[javatest.batch] Completed running 226 tests.
[javatest.batch] Number of Tests Passed      = 226
[javatest.batch] Number of Tests Failed      = 0
[javatest.batch] Number of Tests with Errors = 0

Test name: jpa_core_annotations
[javatest.batch] Completed running 902 tests.
[javatest.batch] Number of Tests Passed      = 902
[javatest.batch] Number of Tests Failed      = 0
[javatest.batch] Number of Tests with Errors = 0

Test name: jpa_core_basic
[javatest.batch] Completed running 12 tests.
[javatest.batch] Number of Tests Passed      = 12
[javatest.batch] Number of Tests Failed      = 0
[javatest.batch] Number of Tests with Errors = 0

Test name: jpa_core_cache
[javatest.batch] Completed running 20 tests.
[javatest.batch] Number of Tests Passed      = 20
[javatest.batch] Number of Tests Failed      = 0
[javatest.batch] Number of Tests with Errors = 0

Test name: jpa_core_callback
[javatest.batch] Completed running 396 tests.
[javatest.batch] Number of Tests Passed      = 396
[javatest.batch] Number of Tests Failed      = 0
[javatest.batch] Number of Tests with Errors = 0

Test name: jpa_core_criteriaapi_CriteriaBuilder
[javatest.batch] Completed running 930 tests.
[javatest.batch] Number of Tests Passed      = 930
[javatest.batch] Number of Tests Failed      = 0
[javatest.batch] Number of Tests with Errors = 0
```
The ```jpa_core_criteriaapi_CriteriaBuilder``` refers to the tests under  jpa/core/criteriaapi/CriteriaBuilder```.  I forget the syntax but I think that we can also specify multiple test folder names in the "ant runclient" call, which could allow us to combine some smaller test groups (e.g. jpa folder has a lot of tests but some jpa subfolders have only a few tests, so they could be combined).  I'm not sure if making this change would impact how we collect test results.  

With smaller test groups, the timeout could be reduced and then perhaps we could reduce the currently (IMO) too-high-max-jvm-memory settings.

No rush on this but thought I would mention it, in case someone has time to work on it.